allensdk/brain_observatory/behavior/data_objects/base/readable_interfaces/sync_file_readable_interface.py

```diff
@@ -0,0 +1,18 @@
+import abc
+
+from allensdk.brain_observatory.behavior.data_objects import DataObject
+
+
+class SyncFileReadableInterface(abc.ABC):
```

Comment:
it might be a little more clear if this and all other readable interfaces are in one file:
`readable_interfaces/abcs.py`
or not? just seems like a lot of tiny files.